ruby/.rubocop.yml

```diff
@@ -1,12 +1,43 @@
-# Commonly used screens these days easily fit more than 80 characters.
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - "bin/*"
+    - "config/**/*"
+    - "db/**/*"
+    - "Guardfile"
+    - "Rakefile"
+
 Metrics/LineLength:
   Max: 120
-
-# Too short methods lead to extraction of single-use methods, which can make
-# the code easier to read (by naming things), but can also clutter the class
 Metrics/MethodLength:
   Include: ["app/controllers/*"]
   Max: 20
-
+Metrics/AbcSize:
+  Exclude: ["app/controllers/*"]
+  Max: 20
+Metrics/AbcSize:
```

Comment:
The duplication of **Metrics/AbcSize:**  gives an _error: duplicate key_ 
also, when running
```console
rubocop
``` 
It mentions:

```console
.rubocop.yml:15: `Metrics/AbcSize` is concealed by line 18
```

Reply:
👍